packages/fast-development-site-react/src/components/site/index.tsx

```diff
@@ -314,7 +337,7 @@ class Site extends React.Component<ISiteProps & IManagedClasses<ISiteManagedClas
             detailViewComponentData: this.getDetailViewComponentData(),
             formView: true,
             devToolsView: false,
-            locale: "en",
+            locale: "English",
```

Comment:
this is not a valid locale, valid locales must start with two lowercase letters indicating the language, which can then be followed by various other parameters. See: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Language for how this is used in a header and https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes for a list of the accepted language values